### PR TITLE
Implementation of mergeDown attribute.

### PR DESCRIPTION
**Pull request by huguesf**
The main purpose of this pull request is to add rowspan functionality (aka ss:MergeDown).
As a side effect, it also add the ss:Index attribute.
Also remove empty style and introduce GitHub Actions (CI).

### CODE REVIEW

**Comment by avielg** [comment]
Nice work! ❤️
Couple nits like whitespace but nothing that should stop this PR!

**Comment by huguesf** [comment]
Hey! Great! Happy you feel it is good.
Could you give me more info about the whitespace thing?

File: Sources/ExcelExport/ExcelExport.swift [hunk 1]
```diff
@@ -165,14 +172,20 @@ public class ExcelExport {
...
         var sheetsValues = [String]()
+        var remainingSpan = [RemainingSpan]()
         for sheet in sheets {
             
             // build sheet
+            var vIndex = 0
             var rows = [String]()
             for row in sheet.rows {
-
+
```
Comment (avielg) [inline Sources/ExcelExport/ExcelExport.swift:186]: @huguesf this one - as I said super nit so not worth blocking this...

Comment (huguesf) [reply Sources/ExcelExport/ExcelExport.swift:186]: :+1:

**Pull request merged**